src/planner/binder/statement/bind_pragma.cpp

```diff
@@ -20,6 +20,27 @@ BoundStatement Binder::Bind(PragmaStatement &stmt) {
 		throw BinderException("PRAGMA function does not have a function specified");
 	}
 
+	QueryErrorContext error_context(root_statement, stmt.stmt_location);
+	// bind and check named params
+	for (auto &kv : stmt.info->named_parameters) {
+		auto entry = bound_function.named_parameters.find(kv.first);
+		if (entry == bound_function.named_parameters.end()) {
+			// create a list of named parameters for the error
+			string named_params;
```

Comment:
Can this be unified with the table function printing code?

Reply:
Sure, this is pretty much a copy-paste of some code in bind_table_function.cpp.

Where should I place this unified piece of code? I'd say function.cpp would be nice, but there's some stuff related to the binder in this piece of code, so perhaps somewhere in the binder?